### PR TITLE
KAN - 13 Add tooltips to the create-update card-type forms

### PR DESCRIPTION
**Pull request by ErickGS28**
### Feature / Bug Description
Add tooltips to the create-update card-type forms
### Solution
Use the [Tooltip component](https://ant.design/components/tooltip) from Ant Design
### Notes
- n/a 

### Tickets
- [TICKET](https://one-sm.atlassian.net/browse/KAN-13)

### Screenshots / Screencasts
image
image

### CODE REVIEW

**Comment by ErickGS28** [comment]
Accidentally, I added the files from KAN-12, but I removed them.

File: src/pages/priority/components/RegisterPriorityForm.tsx [hunk 1]
```diff
@@ -1,67 +0,0 @@
-import { Form, FormInstance, Input, InputNumber } from "antd";
-import Strings from "../../../utils/localizations/Strings";
-import { BsCardText } from "react-icons/bs";
```
Comment (imadiaz) [inline src/pages/priority/components/RegisterPriorityForm.tsx:3]: Why did you delete this?

File: src/pages/priority/components/UpdatePriorityForm.tsx [hunk 2]
```diff
@@ -1,100 +0,0 @@
-import { Form, FormInstance, Input, InputNumber, Select } from "antd";
```
Comment (imadiaz) [inline src/pages/priority/components/UpdatePriorityForm.tsx:1]: also why did you delete this?

Comment (ErickGS28) [reply src/pages/priority/components/UpdatePriorityForm.tsx:1]: I deleted the UpdatePriorityForm.tsx and RegisterPriorityForm.tsx because those files are from the KAN-12 (proriry tooltips) but I accidentally added them to this KAN so I removed them from this pull request

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+### Feature / Bug Description
+Describe the feature / bug. If it's a bug, explain the root cause and how to reproduce.
+
+### Solution
+Describe your changes and explain the technical solution.
```
Comment (imadiaz) [inline .github/pull_request_template.md:5]: why did you change this?

File: src/pages/priority/components/RegisterPriorityForm.tsx (hunk 1 above)
Comment (ErickGS28) [reply src/pages/priority/components/RegisterPriorityForm.tsx]: That file is not more in this PR, it is a file from KAN-12 (priority) I fixed the PR doing the rebase as we talked. Let me know if I have to do any other thing to fix the PR, I just see the 4 files changed.

File: .github/pull_request_template.md (hunk 3 above)
Comment (ErickGS28) [reply .github/pull_request_template.md:5]: I don't remember changing this file, I think it's from when the PR template didn't load and I had to pull it after the change you made. Since then the template change was added to my comm

**Review by imadiaz — APPROVED** [review]

**Review by Isaacjbar — APPROVED** [review]
